pkg/reconciler/v1alpha1/streamingprocessor/streamingprocessor.go

```diff
@@ -57,11 +61,14 @@ const (
 type Reconciler struct {
 	*reconciler.Base
 
+	kedaClientSet kedaclientset.Interface
```

Comment:
let's treat the Keda clientset the same as other clientsets by making it a property on the controller. This will aid in testing as we can provide stub clientsets in a consistent way that record actions that can be asserted.

See
- https://github.com/projectriff/system/blob/master/cmd/controller/main.go#L133-L136
- https://github.com/projectriff/system/blob/master/pkg/reconciler/v1alpha1/streamingprocessor/streamingprocessor_test.go#L66-L70
- https://github.com/projectriff/system/blob/master/pkg/reconciler/v1alpha1/testing/listers.go

Reply:
Implemented via 2515deb